docs/products/caching.md

```diff
@@ -1,11 +1,10 @@
 ---
-title: Aiven for Redis®*
+title: Aiven for Caching
 ---
 
-Aiven for Redis®\* is a fully managed **in-memory NoSQL database**,
+Aiven for Caching is a fully managed **in-memory NoSQL database**,
```

Comment:
can we add a bit about the name change by link to our blog post https://aiven.io/blog/aiven-for-redis-becomes-aiven-for-caching

Reply:
Done